### PR TITLE
feat: "delete last" time entry

### LINKED ISSUE #229: clockify-cli delete last
Sometimes I enter the wrong information and I want to quickly `delete last`  instead of entering the id. This would be intuitive since there is a `clone last`.

### PR DESCRIPTION
**Pull request by jjnilton**
Attempt to close #229, and small details in the CONTRIBUTING.md file.

I'm new to Golang, so any feedback is welcome.

Note: I was not able to build the project on Windows using `make`, I had to do it manually with `go build`. It seems it's because the way the shell variables are set in the Makefile. Also, `grep` and `awk` seem to be required for `make help`, so maybe it's worth mentioning these things somewhere.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -2,18 +2,18 @@
...
-We accept pull requests for bug fixes and features (preferably that were discussed on a issue
+We accept pull requests for bug fixes and features (preferably that were discussed on an issue
 before). Also opening issues with feature requests and reporting bugs are very important
 contributions.
 
 Please do:
 
-- Check in the [issues][issues] if the [bug][bugs] or [feature request][features] has not been
-  submitted.
+- Check in the [issues](https://github.com/lucassabreu/clockify-cli/issues) if the [bug][bugs]
```
Comment (lucassabreu) [inline CONTRIBUTING.md:11]: add a `[issues]` referente to the end of the file instead
```md
[issues]: https://github.com/lucassabreu/clockify-cli/issues
```

File: pkg/cmd/time-entry/delete/delete.go [hunk 2]
```diff
@@ -40,9 +40,17 @@ func NewCmdDelete(f cmdutil.Factory) *cobra.Command {
...
+			# deleting the last time entry
+			$ %[1]s last
+			# no output
+
 			# deleting multiple time entries
 			$ %[1]s 62b5b51085815e619d7ae18d 62b5d55185815e619d7af928
 			# no output
+
+			# deleting last two entries
+			$ %[1]s last last
```
Comment (lucassabreu) [inline pkg/cmd/time-entry/delete/delete.go:52]: i think this is a little bit dangerous

Comment (lucassabreu) [reply pkg/cmd/time-entry/delete/delete.go:52]: i understand it works... but i wouldn't encourage it

**Review by lucassabreu — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (jjnilton) [reply CONTRIBUTING.md]: Done. I completely forgot about this way of linking.

File: pkg/cmd/time-entry/delete/delete.go (hunk 2 above)
Comment (jjnilton) [reply pkg/cmd/time-entry/delete/delete.go]: Got it. Removed that part.

**Pull request merged**